pkg/controller/backup-operator/util.go

```diff
@@ -46,3 +46,15 @@ func isPeriodicBackup(ebSpec *api.BackupSpec) bool {
 	}
 	return false
 }
+
+func containsString(slice []string, s string, modifier func(s string) string) bool {
```

Comment:
Since we're not really using the modifier argument anywhere right now, probably better to simplify this function to a simpler `func containsString(slice []string, s string) bool` and add the modifier later when we actually need it.

Reply:
ack will do.